<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaybackCtrl</class>
 <widget class="QWidget" name="PlaybackCtrl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>38</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <property name="spacing">
    <number>12</number>
   </property>
   <property name="leftMargin">
    <number>24</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButtonPlay">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../res/src.qrc">
         <normaloff>:/img/playback_start.png</normaloff>:/img/playback_start.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonPause">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../res/src.qrc">
         <normaloff>:/img/playback_pause.png</normaloff>:/img/playback_pause.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonStop">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../res/src.qrc">
         <normaloff>:/img/playback_stop.png</normaloff>:/img/playback_stop.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonReverse">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../res/src.qrc">
         <normaloff>:/img/playback_rewind.png</normaloff>:/img/playback_rewind.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButtonPreviousFrame">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../res/src.qrc">
         <normaloff>:/img/playback_step_prev.png</normaloff>:/img/playback_step_prev.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonNextFrame">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../res/src.qrc">
         <normaloff>:/img/playback_step_next.png</normaloff>:/img/playback_step_next.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButtonSlow">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../res/src.qrc">
         <normaloff>:/img/playback_slow.png</normaloff>:/img/playback_slow.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonFast">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../res/src.qrc">
         <normaloff>:/img/playback_fast.png</normaloff>:/img/playback_fast.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButtonPrevious">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../res/src.qrc">
         <normaloff>:/img/playback_prev.png</normaloff>:/img/playback_prev.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonNext">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../res/src.qrc">
         <normaloff>:/img/playback_next.png</normaloff>:/img/playback_next.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../res/src.qrc"/>
 </resources>
 <connections/>
</ui>
